cassandra/src/main/scala/filodb.cassandra/columnstore/TimeSeriesChunksTable.scala

```diff
@@ -126,6 +126,22 @@ sealed class TimeSeriesChunksTable(val dataset: DatasetRef,
     Observable.fromFuture(futRawParts).flatMap { it: Iterator[RawPartData] => Observable.fromIterator(it) }
   }
 
+  def scanPartitionsBySplit(tokens: Seq[(String, String)]): Observable[RawPartData] = {
```

Comment:
Add start and end times to allow for filtering here too

Reply:
We use FilteredPartitionScan sparingly and only in tests without supporting filters - this should be a different PR with different priority.